### ISSUE TITLE
Dockerfile security issues

### ISSUE THREAD

**Issue by ToBeReplaced** [odoo/docker#37]
I've been using my own dockerfile for the past year, and was delighted to see that Odoo has released an official version!

I'd love to migrate over, but there are a few outstanding issues related to security that prevent me from doing so.

First, could we add checksums for the two insecure http downloads? `wkhtmltox.deb` and `odoo.deb` are not verified, and they are downloaded in-the-clear.

Second, could we download all required packages directly from debian and then run odoo completely unprivileged?

For example, in my dockerfile, I install all of:
````
babel
libxslt-python
pyparsing
python-dateutil
python-decorator
python-docutils
python-feedparser
python-imaging
python-jinja2
python-ldap
python-lxml
python-mako
python-mock
python-openid
python-passlib
python-psutil
python-psycopg2
python-reportlab
python-requests
python-simplejson
python-unittest2
python-vobject
python-werkzeug
python-yaml
pytz
````

Then, I create an unprivileged user, do a git checkout of the commit I am happy with, install it in a virtualenv that has access to the system packages, and launch it. Odoo never sees root.

Thanks for embracing docker!

**Comment by sle-odoo** [comment]
Hello,
Thanks for your feedback.



> First, could we add checksums for the two insecure http downloads? wkhtmltox.deb and odoo.deb are not verified, and they are downloaded in-the-clear.

It is true that we do not check the deb file, but we [correctly check wkhtmltopdf](https://github.com/odoo/docker/blob/master/9.0/Dockerfile#L16).



> Second, could we download all required packages directly from debian and then run odoo completely unprivileged?

It is the case, we [run the process with the odoo user](https://github.com/odoo/docker/blob/master/9.0/Dockerfile#L49) created by [the deb](https://github.com/odoo/odoo/blob/9.0/debian/postinst#L14) and all the dependencies are downloaded from debian, [also thanks to the deb](https://github.com/odoo/odoo/blob/9.0/debian/control#L16).

**Comment by ToBeReplaced** [comment]
Ah yes, thanks. Reviewing the .deb is sufficient for my requirements so long as it is pegged (we add a checksum). I missed that for wkhtmltopdf; thanks.

So now the only reason this issue is open is because of wanting a check on `odoo.deb`.

**Comment by aab-odoo** [comment]
Fixed by ad90fca59c9ac52f4f29a195535a3f8ebdddd30d.

Regards
